src/react-components/src/components/Image/Image.tsx

```diff
@@ -34,5 +38,23 @@ export default function Image(props: ImageProps) {
         alt: isDecorative ? "" : altText,
     };
 
-    return <img {...imageProps} />;
+    return (
+        <>
+            <img {...imageProps} />
+            {(imageCaption || imageCredit) && (
+                <div className={bem(image_base_class, [], "annotation")}>
```

Comment:
In the event that this has an image caption and/or credit, instead of a div this should become a `figcaption` and the entire component should be wrapped in a `figure`

Reply:
I updated to use `figure` and `figcaption`. I _think_  up to two `figcaptions` can be included in a `figure` as [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) says "<figcaption> element must be its first or last child.". Is that the structure you had in mind?

Reply:
What we did on the other project was wrap the two divs in the figcaption, so you ended up with something like this:

```
figure
  figcaption
    div1, caption
    div2, credit
```